dart-generator/src/dart-client.ts

```diff
@@ -39,21 +40,21 @@ import 'package:sdkgen_runtime/http_client.dart';
   }
 
   code += `class ApiClient extends SdkgenHttpClient {
-  ApiClient(String baseUrl) : super(baseUrl, _typeTable, _fnTable, _errTable);
+  ApiClient(String baseUrl, http.Client? client) : super(baseUrl, client, _typeTable, _fnTable, _errTable);
```

Comment:
```suggestion
  ApiClient(String baseUrl, [http.Client client]) : super(baseUrl, client, _typeTable, _fnTable, _errTable);
```

Assim é melhor pois não quebra nenhum código existente, não precisa passar null explicitamente